generated/nifgen/nifgen_service.h

```diff
@@ -184,4 +184,9 @@ class NiFgenService final : public NiFgen::Service {
 
 } // namespace nifgen_grpc
 
+namespace nidevice_grpc {
```

Comment:
Ah, yeah. So didn't hit this diff with ni-fake because it was already generating this section for some other custom_type conversions already. It's considering the new enum param (I bet because it's coming from nidevice_grpc package) as a custom type so in this determination [here](https://github.com/ni/grpc-device/blob/ff2901836e4c46c37f34b51f599fb2ac18b4d4a2/source/codegen/templates/service.h.mako#L128) it thinks it should produce this section because it will need to have custom conversion code.

I made a tech debt to look into and fix this: [AB#2192495](https://dev.azure.com/ni/DevCentral/_workitems/edit/2192495)